webapp/channels/src/components/admin_console/team_channel_settings/convert_confirm_modal.tsx

```diff
@@ -101,9 +113,14 @@ const messages = defineMessages({
     toPrivateMessage: {
         id: 'admin.team_channel_settings.convertConfirmModal.toPrivateMessage',
 
-        // This eslint-disable comment can be removed once this component no longer uses FormattedMarkdownMessage
         // eslint-disable-next-line formatjs/no-multiple-whitespaces
```

Comment:
i think we should be able to remove this as well

Reply:
Done!